website/docs/installation/linux-install/install-on-rhel10.md

```diff
@@ -0,0 +1,47 @@
+---
+sidebar_position: 3
+title: Installing on RHEL 10
+description: Install Podman Desktop on a RHEL 10 machine
+keywords: [podman desktop, podman, install, RHEL 10]
+tags: [install-on-rhel-10, podman-desktop-installation]
+---
+
+# Installing on RHEL 10
+
+You can use the subscription manager package to install Podman Desktop on a Red Hat Enterprise Linux (RHEL) 10 machine.
+
+#### Prerequisites
+
+- A RHEL 10 machine.
+- Register with the subscription manager using either your [account details](https://docs.redhat.com/en/documentation/red_hat_enterprise_linux/9/html/configuring_basic_system_settings/assembly_registering-the-system-and-managing-subscriptions_configuring-basic-system-settings#registering-a-system-by-using-the-command-line_assembly_registering-the-system-and-managing-subscriptions) or the [activation key](https://docs.redhat.com/en/documentation/subscription_central/1-latest/html/getting_started_with_rhel_system_registration/basic-reg-rhel-cli#proc-reg-rhel-rhc-act-key_).
+
+#### Procedure
+
+1. Open a terminal, and enable the RHEL extensions repository:
+
+   ```sh
+   $ sudo subscription-manager repos --enable rhel-10-for-amd64-extensions-rpms
```

Comment:
I don't think it's amd64 there, isn't it something different for amd64 ?

Reply:
done